src/injected/visualization/focus-indicator-creator.ts

```diff
@@ -159,10 +175,7 @@ export class FocusIndicatorCreator {
         return focusIndicator;
     };
 
-    private shouldBreakGraph(items: TabbedItem[], curElementIndex: number): boolean {
-        return (
-            curElementIndex === 0 ||
-            items[curElementIndex - 1].tabOrder !== items[curElementIndex].tabOrder - 1
-        );
+    private shouldBreakGraph(item: TabbedItem, prevItem: TabbedItem): boolean {
+        return prevItem == null || prevItem.tabOrder !== item.tabOrder - 1;
```

Comment:
This change looks great to me. I seem to remember there being quite a few places where we referenced the order of the items in the array instead of the items' tab order (for example, the old implementation of this method). Have all of those instances been removed with this change?

Reply:
Somewhat. A fair bit of those were around the `shouldRedraw` method and those are simplified but not totally gone.